test/jdk/java/nio/channels/unixdomain/SocketOptions.java

```diff
@@ -72,7 +72,7 @@ static void testPeerCred() throws Exception {
                 // Check returned user name
 
                 if (!s1.equals(s2)) {
-                    throw new RuntimeException("wrong username");
+                    throw new RuntimeException("wrong username, we got " + s1 + " but property user.name is " + s2);
                 }
```

Comment:
It might be better to use "expected" and "actual" instead of "we got" :-)

I assume you'll bump the copyright header before integrating.

Reply:
Thanks, I adjusted the header and also the output.
Added a line break.